packages/app/src/systems/Account/machines/unlockMachine.tsx

```diff
@@ -84,19 +91,22 @@ export const unlockMachine = createMachine(
       },
       done: {
         type: 'final',
-        data: (_, e: { data: WalletUnlocked }) => e.data,
+        data: (_, e: { data: WalletUnlocked | WalletManager }) => e.data,
       },
     },
   },
   {
     services: {
-      unlock: FetchMachine.create<AccountInputs['unlock'], WalletUnlocked>({
+      unlock: FetchMachine.create<UnlockInput, WalletUnlocked | WalletManager>({
         showError: false,
         maxAttempts: 1,
         async fetch({ input }) {
           if (!input || !input?.password) {
             throw new Error('Password is required to unlock wallet');
           }
+          if (input.manager) {
+            return AccountService.unlockVault(input);
+          }
           return AccountService.unlock(input);
         },
       }),
```

Comment:
can we create other service / event for `unlockManager` ?

who needs the `manager` fires event `UNLOCK_MANAGER`.
who needs the `wallet` fires event `UNLOCK_MANAGER`

Reply:
after doing it can also remove the `input.manager: boolean`